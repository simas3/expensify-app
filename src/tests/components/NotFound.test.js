import React from 'react'
import ReactShallowRenderer from 'react-test-renderer/shallow'
import NotFound from '../../components/NotFound'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json';

test('should render Drawer component', () => {
    const wrapper = shallow(<NotFound />)
    expect(toJson(wrapper)).toMatchSnapshot()
})