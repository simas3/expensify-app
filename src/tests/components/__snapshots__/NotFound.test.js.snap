// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Drawer component 1`] = `
<div>
  Not Found
  <Link
    replace={false}
    to="/"
  >
    Go Home
  </Link>
</div>
`;
